src/test/java/net/mcreator/integration/ui/DialogsTest.java

```diff
@@ -209,6 +205,23 @@ public class DialogsTest {
 				false));
 	}
 
+	@Test public void testUsagesSearchDialogs() throws Throwable {
+		UITestUtil.waitUntilWindowIsOpen(mcreator,

```

Comment:
Wby js exactly the same test called so many times?

Reply:
Because I forgot it's the same method :man_facepalming:

Reply:
:heavy_check_mark: Fixed